doc/faq.rst

```diff
@@ -112,6 +112,34 @@ two solutions:
    processes and one writes a file in the image that another is reading or
    writing).
 
+:code:`ch-image` fails with "certificate verify failed"
+-------------------------------------------------------
+
+When :code:`ch-image` interacts with a remote registry (e.g., via :code:`push`
+or :code:`pull` subcommands), it will verify the registry's HTTPS certificate.
+If this fails, :code:`ch-image` will exit with the error "certificate verify
+failed".
+
+This situation tends to arise with self-signed or institutionally-signed
+certificates, even if the OS is configured to trust them, because on many
+platforms, Requests, the Python HTTP library we use, `includes its own CA
+certificates bundle
+<https://docs.python-requests.org/en/master/user/advanced/#ca-certificates>`_,
+which may differ from that installed on the host system.
```

Comment:
This feels like a lot of commas, but it's probably fine.

Reply:
Good point, let me revise.